tests/unit/test_power_grid_model.py

```diff
@@ -149,10 +149,8 @@ def test_simple_power_flow(model: PowerGridModel, sym_output):
     compare_result(result, sym_output, rtol=0.0, atol=1e-8)
 
 
-def test_simple_permanent_update(
-    model: PowerGridModel, update_batch, sym_output_batch
-):  # error if permanent update is not single scenario
-    model.update(update_data=update_batch)  # single permanent model update
+def test_simple_permanent_update(model: PowerGridModel, update_batch, sym_output_batch):
+    model.update(update_data=get_dataset_scenario(update_batch, 0))  # single permanent model update
```

Comment:
I think it was discussed but how did this become only an OSX problem though? 🤔 
We shouldn't be able to call a .update with a batch data.

Reply:
> I think it was discussed but how did this become only an OSX problem though?

My guess is that this is going to be something related with the alignment or something similar, that is exclusive to Mac OS, and even worse, it may be dependent on the architecture (hence why it passes here and not on the feedstock CI). But that is just my guess. 

> We shouldn't be able to call a .update with a batch data.

It was discussed that we should be able to (I don't think we should, but that is another discussion), and the core actually supports it (just uses the first scenario) as it is shown by the tests added.

I am expecting this to be fine on the feedstock CI, and if it is, then we can (at least for now) not support being able to use batch datasets for the permanent updates, until we figure out what is really going on. But I have the feeling it is going to be nasty to find out...

Reply:
How about testing both original and before then if that was intended to work?

Reply:
What do you mean?